### ISSUE TITLE
Where is /content folder?

### ISSUE THREAD

**Issue by distributev** [distributev/the-app#69]
I look inside theapp.zip 

Where is /content folder? It's inside _internal/../..?

In the zip file it should come directly under TheApp top folder and next to the bin folder

User can add new *.md files to the /content folder and then run /bin/hexo.bat commands on that content. The source_dir of hexo should point to this /content folder.

During development you could have the /frontend/content folder structure next to the /client and /server - _.md files under /content are source files so it's better to move /content/_.md files outside theapp-template folder and next to the other source files /client and /server inside in their own /frontend/content folder structure.

**Comment by luismanuel001** [comment]
Yes it is on the root of hexo, on /_internal/app/frontend-hexo/frontend/content.  It is there because of the issue I mentioned to you early when I started the project, that made hexo generate fail to process any files when the public_dir was set to anything one level above its folder hierarchy (anything starting with "../").  From reading some issues on the hexo repo, I think this is a hexo issue and we could not do much to resolve this.

I'm not sure how you want me to proceed here, I could file an issue on the hexo repo to see if I can get help there, but that could probably take long.

As a different option please check https://github.com/greg-js/hexo-cli-extras.  We can use this plugin to make full use of the hexo.bat script to enhance the user interaction with the hexo content.

With hexo-cli-extras, one can run "/bin/hexo edit" and the command line interface will let you choose a page/post to edit which will open on the user's OS default text editor.

**Comment by distributev** [comment]
1. Yes please file an issue with hexo repo. Put the link to the new issue here so I can follow up myself
2. Meanwhile rename back /_internal/app/frontend-hexo/frontend/content to the original /_internal/app/frontend-hexo/frontend/source and remove the custom _config.yml source_dir change.

Since I cannot get the benefit of having /content/ top level folder then I prefer keeping the default hexo configuration (as much as possible). Later on this might be improved but it depends (again) on functionality which is yet to come.

**Comment by luismanuel001** [comment]
Filed the issue: https://github.com/hexojs/hexo/issues/2053
